testing/tests/ui/excessive_nesting.stderr

```diff
@@ -1,4 +1,4 @@
-error: failed to parse template source at row 14, column 34 near:
+error: failed to parse template source near 14:34
```

Comment:
I find this wording very confusing, in particular `near 14:34`. It doesn't follow the Rust "standard error output" and unless you know what it is already, it's hard to guess. I think the full sentence was better here. The alternative would be to retrieve the file name and then generate the file location like we do for the rest.

Reply:
Fair -- I killed the second commit.